examples/examples.py

```diff
@@ -128,6 +144,24 @@ def stft_plot(signal, sample_rate=SAMPLE_RATE):
 ipd.Audio(drum_out, rate=vca.sample_rate)
 # -
 
+# Additionally, the Drum class can take any number of oscillators.
+
+# +
+my_drum = Drum(
+    pitch_adsr=ADSR(0.25, 0.25, 0.25, 0.25, alpha=3),
+    amp_adsr=ADSR(0.25, 0.25, 0.25, 0.25),
+    vco=[
+        SquareSawVCO(shape=0, midi_f0=23.95, mod_depth=12),
+        SquareSawVCO(shape=0, midi_f0=24.05, mod_depth=12)
+    ],
```

Comment:
As I say, we may just want to fix these to two oscillators. And we might want a mix knob between the two of them. At the moment it just does an average of all oscillators.